apps/browser/src/autofill/utils/collect.ts

```diff
@@ -0,0 +1,477 @@
+import { TYPE_CHECK } from "../constants";
+import { FillableControl, ElementWithOpId, FormElement } from "../types";
+
+/**
+ * Clean up the string `s` to remove non-printable characters and whitespace.
+ * @param {string} s
+ * @returns {string} Clean text
+ */
+function cleanText(s: string | null): string | null {
+  let sVal: string | null = null;
+  s && ((sVal = s.replace(/^\\s+|\\s+$|\\r?\\n.*$/gm, "")), (sVal = 0 < sVal.length ? sVal : null));
+
+  return sVal;
+}
```

Comment:
```suggestion
function cleanText(s: string | null): string | null {
  if (!s) {
    return null;
  }

  const sVal: string | null = s.replace(/^\\s+|\\s+$|\\r?\\n.*$/gm, "");
  if (!sVal.length) {
    return null;
  }


  return sVal;
}
```

Nit, I'm just going to make suggestions for how to rewrite working logic like this. Not necessarily going to do that for everything, but simple ones like this can be likely reworked without issue.

Reply:
Keep in mind we also have prettier formatting enforcement working too, so once we rewrite things like this, we can let prettier do the (opinionated) heavy lifting.